src/components/FeedbackForm.jsx

```diff
@@ -0,0 +1,140 @@
+import axios from 'axios';
+import { useState } from 'react';
+
+export default function FeedbackForm() {
+  const [submitted, setSubmitted] = useState(false);
+  const [valid, setValidity] = useState(false);
+  const [formValues, setFormValues] = useState({
+    firstName: '',
+    lastName: '',
+    mail: '',
+    rating: '',
+    feedback: ''
+  });
+
+  const radioOptions = [
+    { val: "super-bad", emoji: '\u{1F4A9}' },
+    { val: "bad", emoji: '\u{1F922}' },
+    { val: "mediocre", emoji: '\u{1F623}' },
+    { val: "OK", emoji: '\u{1F914}' },
+    { val: "good", emoji: '\u{1F60A}' },
+    { val: "very-good", emoji: '\u{1F60D}' },
+    { val: "excellent", emoji: '\u{1F37A}' },
+  ]
+
+  const changeVal = e => {
+    setFormValues({ ...formValues, [e.target.id]: e.target.value });
+  };
+
+  const radioSelected = e => {
+    setFormValues({ ...formValues, rating: e.target.value });
+  };
+
+  const resetForm = data => {
+    if (data.status === 200) {
+      setFormValues({
+        firstName: '',
+        lastName: '',
+        mail: '',
+        rating: '',
+        feedback: ''
+      });
+      setSubmitted(false);
+      Array.from(document.getElementsByName('rating')).map(item => (item.checked = false));
+    } else {
+      alert('Something went wrong! Please try again.');
+    }
+  };
+
+  const formSubmit = e => {
+    e.preventDefault();
+    setSubmitted(true);
+    const { firstName, lastName, mail, rating, feedback } = formValues;
+    if (firstName && lastName && mail && rating && feedback) {
+      localStorage.setItem('survey-form', JSON.stringify(formValues));
+      setValidity(true);
+      resetForm({ status: 200 });
+      setTimeout(function () {
+        setValidity(false);
+      }, 5000);
+    }
+  };
+
+  return (
+    <div>
+      {valid && <div className="success-msg">Form submitted successfully!</div>}
```

Comment:
i'm a bit confused that "valid" variable is responsible of showing message after form submitted and "submitted" variable is used for showing validation messages. don't you think it would be better to swap them? :)